hudi-hadoop-mr/src/main/java/org/apache/hudi/hadoop/utils/HoodieHiveUtils.java

```diff
@@ -137,4 +139,16 @@ public static HoodieTimeline getTableTimeline(final String tableName, final JobC
     // by default return all completed commits.
     return metaClient.getActiveTimeline().getCommitsTimeline().filterCompletedInstants();
   }
+
+  public static Option<String> getSnapshotMaxCommitTime(JobConf job, String tableName) {
```

Comment:
nit: consider adding javadoc for all public methods (I know we dont follow this consistently, but would be great to add for all new code)